src/win/pty.cc

```diff
@@ -131,6 +131,23 @@ void throw_winpty_error(const char *generalMsg, winpty_error_ptr_t error_ptr) {
   winpty_error_free(error_ptr);
 }
 
+static NAN_METHOD(PtyGetExitCode) {
+  Nan::HandleScope scope;
+
+  if (info.Length() != 1
+    || !info[0]->IsNumber()) // pidHandle
+  {
+    Nan::ThrowError("Usage: pty.getExitCode(pidHandle)");
+    return;
+  }
+
+  uint32_t pidHandle = info[0]->Uint32Value();
+  DWORD exitCode = 0;
+  GetExitCodeProcess((HANDLE)pidHandle, &exitCode);
```

Comment:
@jerch I was a bit lost with getting the integer into a `HANDLE` here tbh. The conversion done to from C++ -> JS is:

```cpp
HANDLE handle = nullptr;
...
marshal->Set(Nan::New<v8::String>("innerPidHandle").ToLocalChecked(), Nan::New<v8::Number>((int)handle));
```

I wrote that as well so maybe that's not right 😉. I did however test on a 64-bit Windows machine running 64-bit node (I think), I was even hosting 64-bit VS Code and running the right bitness terminal as I just added support for that yesterday. I can try changing it to `int32_t` when I get back to my Windows box.